### ISSUE TITLE
Feature Request: Remove jQuery dependency

### ISSUE THREAD

**Issue by AgentSmith0** [jcubic/jquery.terminal#730]
### I have an idea for a new feature for jQuery Terminal

Removing jQuery as a dependency could improve the loading speed of jquery.terminal.

**Comment by jcubic** [comment]
I don't think it will ever happen, but I can check if you can use some jQuery alternative like [Zepto](https://zeptojs.com/) or [Cash](https://kenwheeler.github.io/cash/).

Also, note that jQuery Terminal is twice as big as jQuery, it would be much bigger without jQuery. The code is much shorter because of using jQuery as a dependency, so it would not load faster because it will probably be twice as big. Also jQuery Terminal without jQuery would require rebranding. Maybe Vanilla Terminal. I don't think that this is something I would like to invest my time in.

The alternative is creating a Vanilla Adapter based on jQuery that uses native DOM API.

In short, I don't see myself rewriting the whole library, just to get rid of jQuery.

**Comment by AgentSmith0** [comment]
I expected something like that. Probably jQuery is a helpful dependency in this case. Anyway, thank you very much for your library!

**Comment by jcubic** [comment]
I was playing with using cash library it says it's 8kB, but in fact, the library is 15kB. I've created a clone of jQuery because I needed $.Callbacks that is used in jQuery Terminal. The output file is as small as 21.7kB.

https://github.com/jcubic/jquery/blob/main/cash.min.js

But still, the library doesn't work because jQuery Terminal uses jQuery only CSS selectors like `:visible`, it may take a while to fix all the missing pieces to use Cash. Note that jQuery is only 36kB.

I don't this that 36kB is that much larger to make it worth spending much time working on this.

**Comment by AgentSmith0** [comment]
I now also think like that. Thank you for your efforts!
